app/src/containers/ListOfNews/index.js

```diff
@@ -45,27 +43,30 @@ class ListOfNews extends React.Component {
     }
 
     getNewsListAPI(callback) {
-        const url = new URL(window.location.protocol + '//' + window.location.hostname + this.state.portOption + "/api/v1/news");
-
+      const url = new URL(window.location.protocol + '//' + window.location.hostname + this.state.portOption + "/api/v1/news");
         fetch(url, {
             method: 'GET',
             credentials: 'include',
             cors: 'no-cors'
         }).then((response) => response.json())
             .then((responseJson) => {
+                responseJson.sort((a,b) => new Date(a.updatedAt) < new Date(b.updatedAt) ? 1 : -1);
                 this.setState({ newsList: responseJson })
             }).catch((error) => {
                 console.error(error);
             });
 
         callback();
-    }
+
+}
 
 
+   
     componentDidMount() {
         this.getNewsListAPI(() => {
-            this.setState({ isLoadingPage: false });
```

Comment:
Indentation is important for code readability. We are using tabsize 4 for this project.